pyccel/codegen/printing/fcode.py

```diff
@@ -523,7 +578,7 @@ def _print_AnnotatedComment(self, expr):
         return '!${0} {1}'.format(accel, txt)
 
     def _print_Tuple(self, expr):
-        shape = list(reversed(numpy_shape(expr)))
+        shape = list(reversed(asarray(expr).shape))
```

Comment:
Does a tuple not have a shape?

Reply:
Yes, but I think it does not return the "correct" multi-dimensional shape. This was the problem with using `numpy.shape(expr)`,which was searching for `expr.shape` and returning it.

Reply:
This should be fixed when issue #328 is fixed as a tuple will always be a 1d object.
What is the "wrong" shape?

Reply:
I think this is used in
```
from numpy import array

x = array(((1, 2, 3), (4, 5, 6)))
```
The argument of `array` is a 1D `PythonTuple`, but we want to convert it to a 2D array and get its shape.

Reply:
Ah sorry I didn't realise I was looking at a sympy tuple here

Reply:
By the way, here we are looking at `_print_Tuple` but the same change was made to `_print_PythonTuple`.

Reply:
No it wasn't
https://github.com/pyccel/pyccel/blob/devel-ast-nodes/pyccel/codegen/printing/fcode.py#L586

Reply:
@EmilyBourne You're right, sorry. We handle that in the `__init__` of `numpyext.Array` directly.